src/TLuaInterpreter.cpp

```diff
@@ -750,21 +757,220 @@ int TLuaInterpreter::getBufferTable( lua_State * L )
 
 int TLuaInterpreter::loadRawFile( lua_State * L )
 {
-    string luaSendText="";
-    if( ! lua_isstring( L, 1 ) )
-    {
-        lua_pushstring( L, "loadRawFile: wrong argument type" );
+    QString fileName;
+    if( ! lua_isstring( L, 1 ) ) {
+        lua_pushstring( L, tr( "loadRawFile: bad argument #1 (replay file, optionally with a path, as string expected, got %1)" )
+                           .arg( luaL_typename(L, 1) ).toUtf8().constData() );
         lua_error( L );
         return 1;
     }
     else
-    {
-        luaSendText = lua_tostring( L, 1 );
+        fileName = QDir::fromNativeSeparators( QString::fromUtf8( lua_tostring( L, 1 ) ) );
+
+    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
+#if defined( TILDE_EXPANSION )
```

Comment:
From what I recall it is described as a _**bash**ism_ -  i.e. the use '~' on its own to refer to the user's home directory {not with standing the expansion to *~otherusername* for other users' home directory} and is not something that users of some other OSs coming from a place called Redmond in the USofA are going to necessarily be familiar with...